guides-and-standards/guide-choosing-software-dependencies.md

```diff
@@ -0,0 +1,59 @@
+  - Hint: you can search for the word "commercial" in the license. This will often take you to text about whether the software can be used commercially. If it can’t, it will usually say something like "for non-commercial use only."
+- **The tool can be run on the command line without human input.** A Graphical User Interface (GUI) (including web applications like Streamlit) is not required to run the tool. The tool isn’t encoded in notebooks.
+- **The tool can be run on a Linux platform.** This is a requirement\* for things dockerization and for running on AWS.
+  - *It’s not actually a*requirement\* — the Mac operating systems can’t really be dockerized, but Windows can. Similarly, there are Windows operating systems on AWS but these are sometimes annoying to use.
+
+### Important — think carefully before proceeding with the tool if these criteria are not met
+
+- **The software code is on GitHub or another popular developer platform like GitLab or BitBucket.** Using these platforms signals that the tool is open source and likely versioned. Sometimes you'll find code on SourceForge. This is ok and potentially workable, but not the best. Sometimes you'll find code in the supplement of a paper. This is not awesome and probably signals that the tool will be hard to install or use but you can still try if you really want to.
+- **The tool is easily installable.** To be able to use a tool, we need to be able to install it. Below I’ve included somewhat arbitrary gradations of what makes something easy to install. You can check these things by eye to quickly see if you think a tool will be installable, but often the best way to determine this is to install and use the tool.
+  - Very easy: cross-platform installation as a conda package
+  - Easy: installable from another package repository with another package manager like pip or poetry (python) or `install_packages()` (R)
+  - Somewhat easy: Installable from GitHub with a package manager
+  - Somewhat easy: Has a dockerfile and an available Docker container
+  - Somewhat difficult: has an environment yaml file or requirements.txt file that lists versioned dependencies and has installation instructions that involve cloning the repository
+  - Difficult: git clone & compile from source
+  - Difficult: No installation instructions
+
+### Nice — but not necessary if you’re willing to do the leg work to make sure the tool works
+
+- **The tool has tests.** Tests ensure the accuracy and quality of a software tool. See [this AUG](https://training.arcadiascience.com/arcadia-users-group/20231104-testing-concepts/lesson/) for more about software tests.
+  - Hint: tests will often be a `test/` or `tests` folder, either in the root of the repository or
+- **The software is maintained.**
+  - Hint: there is a difference in the timestamp between the first and last commit and the last commit occurred recently (~months)
```

Comment:
Maybe also mention here to look for a history of closed PRs and at the "insights" page on GitHub (particularly the "contributors" tab, which shows contributions over time).